autoload/test/python/pytest.vim

```diff
@@ -7,7 +7,7 @@ function! test#python#pytest#test_file(file) abort
     if exists('g:test#python#runner')
       return g:test#python#runner == 'pytest'
     else
-      return executable('py.test')
+      return executable('pytest') || executable('py.test')
```

Comment:
Not sure if keeping the 2nd one makes sense?!

Reply:
If `pytest` will always exist alongside `py.test`, then by all means remove the `py.test` check.